content/rekor-v2-ga.md

```diff
@@ -0,0 +1,166 @@
+Cosign produces are verifiable by the latest Go, Java and Python clients as well. Support for JavaScript and Ruby will follow.
+
+If you have any questions or feedback, please reach out on Slack on the `#rekor` or `#clients` channels, and as you upgrade clients,
+please report any issues on GitHub.
+
+## FAQ
+
+### What's happening with Rekor v1?
+
+Rekor v1 will continue to run in parallel with Rekor v2. Clients using the publicly distributed SigningConfig and TrustedRoot files
+will automatically and seamlessly transition to using Rekor v2 and verifying entries.
+
+We will eventually freeze Rekor v1, disallowing new entry uploads. This will be announced one year in advance as per our deprecation
+guidelines.  Regardless, we strongly encourage users to upgrade to the latest clients as soon as possible.
+
+### What's the new URL for the log?
+
+Previously, Rekor was accessible via a single URL `rekor.sigstore.dev`. This URL abstracted
+log shards, where a new tree is created to upload entries. This added complexity for log operation.
+
+Going forward, each log shard will have a unique URL, e.g. `logYEAR-rev.rekor.sigstore.dev`, which matches
+how Certificate Transparency handles log deployments.
+Log shard URLs will be distributed via Sigstore's TUF repository. You can find the latest URL
+by inspecting the [SigningConfig](https://github.com/sigstore/root-signing/blob/main/targets/signing_config.v0.2.json).
+
+If you're building a client, you MUST NOT hardcode this URL. We will periodically rotate the log instance, and while
+we'll publicly communicate when we spin up a new log, we will also freeze the previous log instance shortly afterwards.
+
+### How do I upload entries to the public instance?
+
+The easiest way is to use an updated Sigstore client or SDK.
+
+If you want to experiment with the API, we've provided an
+[example](https://github.com/sigstore/rekor-tiles/blob/main/CLIENTS.md#rekor-v2-the-bash-way). Note
+that Rekor v2 exposes only a single endpoint, `/api/v2/log/entries`.
+
+### How do I read entries from the public instance?
+
+If you're monitoring the log, the easiest way is to use [rekor-monitor](https://github.com/sigstore/rekor-monitor).
+
+Read requests are served by a standardized [read API](https://github.com/C2SP/C2SP/blob/main/tlog-tiles.md#merkle-tree)
+that serves tiles and entries. Note that these APIs are significantly different than Rekor v1, namely
+there are no Get-By-Log-Index or Get-By-Leaf-Hash APIs. The Sigstore clients handle the logic to fetch the set of tiles
+needed to compute an inclusion proof given a log index.
+
+Unless you're monitoring the log for entries, we discourage using the read API for computing inclusion proofs. The log
+will return inclusion proofs on upload and these should be persisted alongside artifacts and their signatures.
+
+Note that the Rekor v2 server does expose a read API, but this is not currently implemented and will only be
```

Comment:
I'm hesitant about this framing - the Rekor v2 specification provides both a read and write API. The Rekor v2 binary only provides a write API, but we implement the read API in another way. Could we try to do one of either 1) gloss over the implementations of read/write APIs, since in the public instance they're both available and abstract, or 2) get more into the details of how the binary service provides a write API and the read API is served via object storage, since this has implications for how private deployers will operate their service. I feel like the current wording takes a middle ground that makes it more confusing.